src/DocFxOpenApi/Helpers/MessageHelper.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Licensed to DocFX Companion Tools and contributors under one or more agreements.
+// DocFX Companion Tools and contributors licenses this file to you under the MIT license.
+
+namespace DocFxOpenApi.Helpers
+{
+    using System;
+    using global::DocFxOpenApi.Domain;
+
+    /// <summary>
+    /// Helper methods to write messages to the console.
+    /// </summary>
+    public class MessageHelper
+    {
+        private readonly CommandlineOptions options;
```

Comment:
same here _ pattern makes things easier to use (no need of the this. for example)

Reply:
fixed